### PR TITLE
Finishing footer component

### PR DESCRIPTION
**Pull request by OmerSchechter**
#187

### CODE REVIEW

**Comment by OmerSchechter** [comment]
Current state:

screen shot 2016-11-21 at 21 37 19

File: client/src/app/components/footer/footer.component.scss [hunk 1]
```diff
@@ -0,0 +1,6 @@
+.container {
+	position: absolute;
```
Comment (nirgn975) [inline client/src/app/components/footer/footer.component.scss]: Don't why, I think it's deleted your indentation.

File: client/src/styles.scss [hunk 2]
```diff
@@ -1,5 +1,10 @@
 /* You can add global styles to this file, and also import other style files */
 
+html, body {
+	height: 100%;
+	position: relative;
```
Comment (nirgn975) [inline client/src/styles.scss]: Indentation

File: client/src/app/components/footer/footer.component.html [hunk 3]
```diff
@@ -1,3 +1,10 @@
-<p>
-  footer works!
-</p>
+<div class="container">
+	<div class="col-xs-6">
```
Comment (nirgn975) [inline client/src/app/components/footer/footer.component.html]: I think you have just one space there.

File: client/src/styles.scss (hunk 2 above)
Comment (nirgn975) [inline client/src/styles.scss]: You need the `html` there? the body isn't enough?

File: client/src/app/components/footer/footer.component.scss (hunk 1 above)
Comment (OmerSchechter) [inline client/src/app/components/footer/footer.component.scss]: Fixed.

File: client/src/styles.scss (hunk 2 above)
Comment (OmerSchechter) [inline client/src/styles.scss]: I do. Take a look at the picture I'm attaching, this is how it looks when the `height: 100%;` is removed from the `html` element.
The `body` element looks at it's parent(`html`) to see what 100% means.
[See this discussion.](http://stackoverflow.com/questions/6654958)
screen shot 2016-11-23 at 19 35 44

**Comment by nirgn975** [comment]
@OmerSchechter Once Travis-CI pass this PR, assign the PR to me, and I'll merge.

File: client/src/app/components/footer/footer.component.scss [hunk 4]
```diff
@@ -0,0 +1,7 @@
+.container {
+  position: absolute;
+  bottom: 0;
+  left: 0;
+  right: 0;
+}
+/* Ignore this - here for Travis*/
```
Comment (nirgn975) [inline client/src/app/components/footer/footer.component.scss]: @OmerSchechter I saw travis works on your repo - great! Now, just remove this text (leave here one empty line at the end of the file).

Comment (OmerSchechter) [inline client/src/app/components/footer/footer.component.scss]: I've reverted the commit.

**Pull request merged**